oauth2-server/src/main/java/com/clouway/oauth2/IssueNewTokenActivity.java

```diff
@@ -57,14 +57,21 @@ public Response execute(Client client, Identity identity, Set<String> scopes, Re
       Pem.Block key = certificates.get(certKey);
       String host = request.header("Host");
 
+      Map<String, Object> claims = new LinkedHashMap<>();
+      claims.put("iss", host);
+      claims.put("aud", client.id);
+      claims.put("sub", identity.id());
+      claims.put("iat", instant.asDate());
+      claims.put("exp", new Date(accessToken.expirationTimestamp()));
```

Comment:
In Java 7's Date class it says that :
A thin wrapper around a millisecond value that allows JDBC to identify this as an SQL DATE value. A milliseconds value represents the number of milliseconds that have passed since January 1, 1970 00:00:00.000 GMT. 

Isnt instant just that - a new Date() ?